Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Install Operating system and dependencies
+FROM ubuntu:20.04
+# To prevent interactive prompts
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update 
+RUN apt-get install -y curl git wget unzip libgconf-2-4 gdb libstdc++6 libglu1-mesa fonts-droid-fallback lib32stdc++6 python3 psmisc
+RUN apt-get clean
+
+# download Flutter SDK from Flutter Github repo
+RUN git clone https://github.com/flutter/flutter.git /usr/local/flutter
+
+# Set flutter environment path
+ENV PATH="/usr/local/flutter/bin:/usr/local/flutter/bin/cache/dart-sdk/bin:${PATH}"
+
+# Run flutter doctor
+RUN flutter doctor
+
+# Enable flutter web
+RUN flutter channel stable
```

Comment:
Just a quick question. Will there ever be a time where it would be better to use `RUN flutter channel master` vs `RUN flutter channel stable`?

Reply:
The master channel represents the latest version of flutter out but not one that is fully tested... so while you might want to use master during development when you are putting something out to production you likely want to use the stable channel...

The only reason I can think that you might use master is if there is a fix in there for something critical for the app and its the only way you can fix a critical issue thats out in production... and even then you want to get back to stable as soon as the fix for you problem has been merged to stable